src/javascript/_common/utility.js

```diff
@@ -179,6 +179,15 @@ const getStaticHash = () => {
     return static_hash;
 };
 
+const upgradePulser = (() => {
```

Comment:
The idea of `_common` folder is to serve shared logics between different sections. Therefore, if a function is only related to a particular section, better to move it there. This also prevents having extra unused code in another section.

Besides, having this here will add jQuery dependency to `app_2`, which is not the plan.

Reply:
ok, noted. since this function is mostly related to new account opening, would move it to `common/account_opening`